paddle/fluid/pybind/op_function.h

```diff
@@ -147,6 +147,61 @@ ConstructDuplicableOutput(const size_t num) {
   }
   return res;
 }
+
+static inline std::shared_ptr<imperative::VarBase> ConstructViewOutput(
+    const std::shared_ptr<imperative::VarBase>& input_var) {
+  PADDLE_ENFORCE_EQ(
+      input_var->Var().IsInitialized(), true,
+      platform::errors::InvalidArgument("Tensor %s has not been initialized!",
+                                        input_var->Name()));
+  PADDLE_ENFORCE_EQ(input_var->Var().IsType<framework::LoDTensor>() ||
+                        input_var->Var().IsType<framework::SelectedRows>(),
+                    true,
+                    platform::errors::InvalidArgument(
+                        "Type of Tensor[%s] must be LoDTensor or SelectedRows!",
+                        input_var->Name()));
+
+  auto tracer = imperative::GetCurrentTracer();
+  auto view_output_var = std::shared_ptr<imperative::VarBase>(
+      new imperative::VarBase(tracer->GenerateUniqueName()));
+  view_output_var->SetPersistable(input_var->Persistable());
+  view_output_var->SetType(input_var->Type());
+  view_output_var->SetDataType(input_var->DataType());
+
+  if (input_var->Var().IsType<framework::LoDTensor>()) {
+    const auto& input_tensor = input_var->Var().Get<framework::LoDTensor>();
+    PADDLE_ENFORCE_EQ(
+        input_tensor.IsInitialized(), true,
+        platform::errors::InvalidArgument(
+            "LoDTensor %s has not been initialized!", input_var->Name()));
+
+    auto* view_output_tensor =
+        view_output_var->MutableVar()->GetMutable<framework::LoDTensor>();
+    view_output_tensor->ShareDataWith(input_tensor);
+    view_output_tensor->ShareInplaceVersionCounterWith(input_tensor);
+  } else {
+    const auto& input_selected_rows =
+        input_var->Var().Get<framework::SelectedRows>();
+    PADDLE_ENFORCE_EQ(
+        input_selected_rows.value().IsInitialized(), true,
+        platform::errors::InvalidArgument(
+            "SelectedRows %s has not been initialized!", input_var->Name()));
+
+    auto* view_output_selected_rows =
+        view_output_var->MutableVar()->GetMutable<framework::SelectedRows>();
+    view_output_selected_rows->set_height(input_selected_rows.height());
+    view_output_selected_rows->set_rows(input_selected_rows.rows());
+    view_output_selected_rows->mutable_value()->ShareDataWith(
+        input_selected_rows.value());
+    view_output_selected_rows->mutable_value()->ShareInplaceVersionCounterWith(
+        input_selected_rows.value());
+  }
+
+  VLOG(3) << "The output Var(" << view_output_var->Name()
+          << ") share allocation with Input Var(" << input_var->Name() << ").";
```

Comment:
Better to keep the letter case of "Output Var" and "Input Var"  as consistant in the log message.

Reply:
done